### PR TITLE
Feat(api-platform): api platform documentation spike

### LINKED ISSUE #32581: Spike : doc - auto-generation ?
As a developer an easy way to document our endpoints so I can provide to other developers a good documentation  

## Description 

We need to know how to (auto)-generate documentation for an endpoint , including scope 

Doc examples : 
[github API Rest](https://docs.github.com/fr/rest/overview/permissions-required-for-fine-grained-personal-access-tokens?apiVersion=2022-11-28#blocking)
[Slack API](https://api.slack.com/legacy/oauth-scopes)

## Attendees

- Documentation : how to (auto-generate) documentation for endpoint / scopes
- If not possible "AS IS" : TS ( technical stories ) to implement it

## Acceptance criterarias

- Lead Tech & PM validation

### PR DESCRIPTION
**Pull request by tleon**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | Spike for the api platform documentation.
| Type?             |  new feature 
| Category?         | BO 
| BC breaks?        |  no
| Deprecations?     |  no
| How to test?      | <ul><li>Create the asset directory `mkdir admin-dev/bundles ` </li> <li>install the assets `bin/console asset:install admin-dev`<li> log in the BO </li><li> go to /new-api/docs</li></ul>
| Fixed ticket?     | Fixes #32581
| Related PRs       | N/A
| Sponsor company   | Prestashop

### CODE REVIEW

**Review by matks — COMMENTED** [review]
CI is red please fix it 😉

File: tools/assets/build.sh [hunk 1]
```diff
@@ -28,6 +28,7 @@ function build {
   npm ci
   npm run build
   popd
+
```
Comment (M0rgan01) [inline tools/assets/build.sh]: No space needed here

File: .gitignore [hunk 2]
```diff
@@ -122,6 +122,9 @@ tests-legacy/.phpunit.result.cache
 !/admin-dev/themes/new-theme/public/theme.rtlfix
 !/admin-dev/themes/default/public/theme.rtlfix
 
+# Api plateform assets
```
Comment (FabienPapet) [inline .gitignore]:
```suggestion
# Api platform assets
```

File: src/PrestaShopBundle/ApiPlatform/Resources/Hook.php [hunk 3]
```diff
@@ -43,6 +43,26 @@
...
+                'description' => 'Get hook status B',
+                'parameters' => [
+                    [
+                        'name' => 'hookId',
+                        'in' => 'path',
+                        'required' => true,
+                        'schema' => [
+                            'type' => 'string',
+                        ],
+                        'description' => 'this is a description of the parameter',
```
Comment (FabienPapet) [inline src/PrestaShopBundle/ApiPlatform/Resources/Hook.php]: 👀

**Review by FabienPapet — CHANGES_REQUESTED** [review]

**Review by FabienPapet — APPROVED** [review]

**Review by M0rgan01 — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Comment by paulnoelcholot** [comment]
Hello @tleon,

I tested your PR and it's OK for me! :tada: 

Thanks!

**Review by paulnoelcholot — APPROVED** [review]

**Pull request merged**